tests/Jira/ApiTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Tests\chobie\Jira;
+
+use chobie\Jira\Api;
+use chobie\Jira\Api\Authentication\Anonymous;
+
+/**
+ * Class ApiTest
+ *
+ * @package Tests\chobie\Jira
+ */
+class ApiTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * Tests that any trailing slash in the endpoint url is removed before being stored in the object state
+     */
+    public function testSetEndpointTrailingSlash()
```

Comment:
While you're at it please also add a test that shows that url is kept as is when no trailing `/` is used.